src/kv/KeyValueDB.h

```diff
@@ -146,6 +146,9 @@ class KeyValueDB {
   virtual int create_and_open(std::ostream &out) = 0;
   virtual void close() { }
 
+  // try to repair K/V database. leveldb and rocksdb require that database must be not opened.
```

Comment:
@Liuchang0812 Thanks for PR.
I believe It would be even better if we start comment with capital letter something as:
// Try to repair K/V database

Also I believe there is no such use of leaving space between `//` and `Try`